code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -221,6 +221,48 @@
 		being_used = 0
 		..()
 
+/obj/item/slimepotion/transference
+	name = "consciousness transference potion"
+	desc = "A strange slime-based chemical that, when used, allows the user to transfer their consciousness to a lesser being."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "bottle19"
+	origin_tech = "biotech=6"
+	var/prompted = FALSE
+	var/animal_type = SENTIENCE_ORGANIC
+
+/obj/item/slimepotion/transference/afterattack(mob/living/M, mob/user)
+	if(prompted || !ismob(M))
+		return
```

Comment:
If you don't have a 3rd argument, `proximity_flag`, and check that it is not 0, then you'll be able to activate the potion from any range.